js/interface.js

```diff
@@ -24,6 +24,16 @@
   var menuDataSources = [];
   var customMenus = [];
   var customMenuLoadingPromise;
+  var previousMenu;
+  var defaultMenuPackages = [
```

Comment:
@tonytlwu you happy with this?

Reply:
@squallstar @AndrRyaz No, I don't want to have to update this whenever we create new menus. Also, it's unclear what `defaultMenuPackages` means (why would it have `com.fliplet.menu.eversheds` in it?

Reply:
@tonytlwu could you please give guidance how can I get a list of unique packages in order to remove this array

Reply:
@AndrRyaz It depends on what `defaultMenuPakcages` means and why you need it.

Essentially, the `GET v1/widgets` API endpoint, which can include `include_all_versions=true` to return all versions of each widget, should be able to help you.

There's already a function that this component is using. Have you seen it and do you understand it? https://github.com/Fliplet/fliplet-widget-menu-manager/blob/03fa9546053d178f7fa8a6f24c423d8e11152d63/js/interface.js#L296

Reply:
@tonytlwu
Yes, I added this query from the very beginning, but I created an array of default widgets in order to filter them by version and display the latest.
The parameter `include_all_versions = true` adds a couple of widget records with different versions to the request, but my question is how can I qualify and filter by the required types of menu widgets if, in addition to the default packages like `com.fliplet-widget-bottom-menu`, there is `com.fliplet-widget-bottom-menu-andrey` which may have a newer version

Reply:
@AndrRyaz It depends on the meaning and purpose of the variable `defaultMenuPackages`. Do you mean a list of system menus i.e. not assigned/custom to any organisation?

Reply:
Yes, default system menus and i removed unnecessary menu type in this commit - https://github.com/Fliplet/fliplet-widget-menu-manager/pull/40/commits/3e96ee3465bf98e9ae300bb7bf9b76945ba54aba

Reply:
@tonytlwu ^

Reply:
@AndrRyaz Once you get all the widgets back, you can filter based on the following rules:

1. If it has an `organizationId`, it's a custom and available to the organisation only
2. If it doesn't have an `organizationId` it's available to all users as a system menu

Reply:
@tonytlwu sorry for a long time answer, i fixed all your points.
Here - https://github.com/Fliplet/fliplet-widget-menu-manager/pull/40/files